### PR TITLE
Clean up remaining parts of expression ABNF to be used in other ABNF

### PR DESCRIPTION
**Pull request by cabo**

### CODE REVIEW

File: draft-ietf-jsonpath-base.md [hunk 1]
```diff
@@ -1149,23 +1149,22 @@ Some examples:
...
-decimal-point = %x2E       ; .
-digit1-9 = %x31-39         ; 1-9
-e = %x65 / %x45            ; e E
-exp = e [ minus / plus ] 1*DIGIT
-frac = decimal-point 1*DIGIT
-jsint = zero / ( digit1-9 *DIGIT )
-minus = %x2D               ; -
-plus = %x2B                ; +
-zero = %x30                ; 0
+number = int [ frac ] [ exp ]
```
Comment (glyn) [inline draft-ietf-jsonpath-base.md:1155]: Parson my ignorance, but where is `int` defined?

**Review by glyn — COMMENTED** [review]
LGTM with one query.

File: draft-ietf-jsonpath-base.md (hunk 1 above)
Comment (cabo) [reply draft-ietf-jsonpath-base.md:1155]: 3.5.4.  Index Selector, second ABNF block

**Review by goessner — APPROVED** [review]

File: draft-ietf-jsonpath-base.md (hunk 1 above)
Comment (glyn) [reply draft-ietf-jsonpath-base.md:1155]: Ah, of course. Thanks.

**Review by glyn — APPROVED** [review]

**Comment by cabo** [comment]
Thanks, the rebase looks good.

**Comment by glyn** [comment]
@cabo After merging some PRs I fixed a conflict here and then added a commit. PTAL and merge if you are happy.

**Pull request merged**